src/components/Dialog.js

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import DefaultButton from './DefaultButton.js';
+import PrimaryButton from './PrimaryButton.js';
+
+Dialog.propTypes = {
+  onDialog: PropTypes.func,
+  message: PropTypes.string,
+  nameOfGame: PropTypes.string
+};
+
+export default function Dialog({ onDialog, message, nameOfGame }) {
+  return (
+    <Background onClick={() => onDialog(false)}>
+      <DialogBox onClick={(e) => e.stopPropagation()}>
```

Comment:
Cool that you added it, its for clicking outside of the dialog and the window disappears, right?
I got crazy with it and left it away in the end😅

Reply:
Yes it is.